css/styles.css

```diff
@@ -517,8 +561,6 @@ textarea {
   padding: 0;
   gap: 10px;
   align-items: center;
-
-  /* background-color: black; */
```

Comment:
- _(optional)_ It's best practice to always keep your code as clean as possible. So please kindly remove all commented lines of code